course_content/cartopy_course/solutions/cartopy_exercise_2

```diff
@@ -0,0 +1,24 @@
+
+# Cartopy course, exercise 2
+
+# Use 'set_global' to work out what it was actually doing earlier.
+
+# Code as previously ...
+proj = ccrs.EquidistantConic()
+x0, y0 = -50, -30
+x1, y1 = 10, 55
+
+ax = plt.axes(projection=proj)
+ax.coastlines()
+plt.plot([x0, x1], [y0, y1], linewidth=4)
+
+# But now added 'set_global()'
+ax.set_global()
+
+plt.show()
+
+#
+# NOTE:
+# Result is a tiny blue dot, close to lat=0, lon=0.
+# Because on this map, the native "X" and "Y" coordinates are in **metres** from the origin.
+#
```

Comment:
This does rely on the person using the solutions, which we can't guarantee

Reply:
But if they don't load the solution, they don't even see this.
We could make this executable, i.e. print the explanation, if you think that helps ??

Reply:
I guess it makes sense to keep here as it is a comment about the provided solution

Reply:
Update: I have since tried this technique of "write observations in a print statement" 
[elsewhere](https://github.com/pp-mo/scitools-courses/blob/1b542850325a08e7f11cace1a074c22ac7046c9f/course_content/iris_course/solutions/iris_exercise_4.1a).
To my surprise, I think it actually looks ok.  What do you think @lbdreyer ?